manifest/sourcefile.py

```diff
@@ -323,6 +349,9 @@ def manifest_items(self):
         elif self.name_is_webdriver:
             rv = [WebdriverSpecTest(self, self.url)]
 
+        elif self.content_is_css_manual:
```

Comment:
(@ms2ger admitted he mistook this for support for visual tests, which indeed would, if done naïvely, get rid of all testharness/reftest tests, so this is resolved)